src/lib/json-helper.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Retrieves a nested value from an object using an array of keys as a path
+ * @template T - The type of the input object
+ * @param {T} obj - The object to traverse
+ * @param {string[]} path - Array of keys representing the path to the desired value
+ * @returns {unknown} The value at the specified path, empty string if the path leads to null/undefined, or the value if found
+ * @example
+ * const obj = { user: { profile: { name: "John" } } };
+ * const path = ["user", "profile", "name"];
+ * const name = getNestedValue(obj, path); // Returns "John"
+ */
+
+export const getNestedValue = <T,>(json: T, path: string[]): unknown => {
```

Comment:
Suggest to add an optional "undefined" return value, defaulting to an empty string. I typically dislike using empty strings on undefined properties, but in a frontend it sometimes is easy